src/MultiRobot.elm

```diff
@@ -0,0 +1,65 @@
+module Main exposing (Model, init, main, subscriptions, update, view)
```

Comment:
You shouldn't expose `main`, it won't be used outside this module.
You should expose `Msg`.